README.md

```diff
@@ -36,6 +36,12 @@ See our [documentation](https://github.com/FourthState/plasma-mvp-rootchain/blob
 5. ``ganache-cli`` // run as a background process
 6. ``npm test``
 
+
+### Running
+The first migration file `1_initial_migration` deploys the `PriorityQueue` library, while the second one `2_deploy_rootchain` actually deploys the `RootChain` contract. Ethereum requires libraries to already be deployed prior to be used by other contracts.
```

Comment:
Can you add:
...`1_initial_migration ` deploys the `PriorityQueue` library and links it to the `Rootchain` and `PriorityQueue_Test` contracts ...

Reply:
I think we should exclude the Test contract. It's not user facing and is only concerned with tests

Reply:
Yeah sure then it'd be 
...`1_initial_migration` deploys the `PriorityQueue` library and links it to the `Rootchain`  contract ...

Reply:
So tbh `link` is not necessarily an Ethereum 101 concept, hence I'd say it's okay to abstract it away. No worries nevertheless, I can update it.